modules/doc/content/ncrc/ncrc_local_binary.md

```diff
@@ -42,144 +42,176 @@ optional arguments:
   -h, --help            show this help message and exit
 ```
 
-## Common NCRC Usage Examples
+## Common Usage
 
-!alert note
-Where applicable, you only need to activate the environment once. The examples below assume the environment has not yet been activated.
+##### List all available NCRC applications
 
--  List all available NCRC applications
+```bash
+ $> ncrc list
 
-  ```bash
-  $> ncrc list
+# Use 'ncrc search name-of-application' to list more detail
+# NCRC applications available:
```

Comment:
Did you mean for this to be the highest level of heading?

Reply:
That happens to be a comment being generated by the `ncrc` script:

```pre
\```bash
 $> ncrc list

# Use 'ncrc search name-of-application' to list more detail
# NCRC applications available:

pronghorn
bison
griffin
sabertooth
relap7
\```
```

Reply:
I've been re-doing the indentation going on. I don't much care for the itemized list justification going on, for only half the page...

Reply:
Oh sorry, I completely missed that it was part of the output..... You can disregard this one. I liked how you moved away from so many itemized lists as well.